Makefile

```diff
@@ -86,3 +86,7 @@ rpm: $(RPM_SPECFILE) $(RPM_TARBALL)
 	rpmbuild -bb \
 		--define "_topdir $(CURDIR)/rpmbuild" \
 		$(RPM_SPECFILE)
+
+.PHONY: swagger-specs
+swagger-specs:
+	swagger generate spec --scan-models --output=docs/rcm-api.yaml
```

Comment:
What is the requirement here? Where does the `swagger` tool come from? What is the recommended way to run this?

I think it is ok to make this a run-always target. A dependency list would easily get out-of-sync, and we cannot magically know which files the swagger tool inspects. So I wouldn't worry about making this more fine-grained.

Reply:
In your case probably AUR. For fedora it should run in a container as mentioned above, but seems to fail. I used homebrew to install it and it ran just fine, it's sad that the container fails.